lol.janjic.Splitcat.yml

```diff
@@ -0,0 +1,35 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: lol.janjic.Splitcat
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: splitcat
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+modules:
+  # FlutterApp
+  - name: FlutterApp
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - "./build-flatpak.sh"
+    sources:
+      - type: file
+        url: https://github.com/vogonwann/splitcat/releases/download/v0.1.4/splitcat-linux-x64.tar.gz
+        sha256: 7d3a7f11c4244f038f670b2017848a9baa8c80cc5ddd4fa47cef3da7336a1232
+      - type: dir
+        path: ../splitcat
```

Comment:
where is it gonna find this on the builders?

Reply:
I'm not sure what you mean...

Reply:
remove these two lines, they are wrong.

Reply:
29 and 30?

Reply:
yes

Reply:
yes

Reply:
done